### DISCUSSION TITLE
23.05.3，4 IPV6问题

### DISCUSSION THREAD

**Discussion by magic888058** [immortalwrt/immortalwrt#1498] [General]
23.05.3，4两个版本与23.05.2版本一样的设置,公网ipv6能正常获取，各个设备端IPV6也是正常获取，网络测试IPV6不是不通，有没有遇到相同情况的，网上教程方案试多个也都是不行

**Comment by wbchn** [comment]
我这里的情况不知道跟你是不是一样:
lan口配置DHCPv6 stateless
配置好，不能访问
重启后，wan口 IPv6 PD / lan 口IPv6 地址均正常，设备端获取IPv6 正常，路由器和设备访问IPv6正常。
再运行一段时间后，wan口 IPv6 PD / lan 口IPv6 地址消失，只剩下 wan_6 ip地址，设备IPv6地址存在，此时路由器和设备无法访问IPv6

现在在排查是不是其他插件引起的。

**Comment by magic888058** [comment]
我的docker 27版本引起的，删掉就正常了
